projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -953,7 +996,7 @@ export class MapOlService {
     }
 
     if (l.bbox) {
-      layeroptions.extent = transformExtent(l.bbox, WGS84, this.map.getView().getProjection().getCode());
+      layeroptions.extent = transformExtent(l.bbox.slice(0, 4) as [number, number, number, number], WGS84, this.map.getView().getProjection().getCode());
```

Comment:
Do we need to `slice` all bboxes?  I think GeoJSON can handle 3D bboxes https://tools.ietf.org/html/rfc7946#section-5

Reply:
You'd think so, right?
But when I include map-ol in a ukis-project (as we currently do in WUEKIS) and try to serve the app, I get

```
ERROR in projects/map-ol/src/lib/map-ol.service.ts:706:45 - error TS2345: Argument of type 'TGeoExtent' is not assignable to parameter of type 'Extent'.
  Type '[number, number, number, number, number, number]' is not assignable to type '[number, number, number, number]'.
    Types of property 'length' are incompatible.
      Type '6' is not assignable to type '4'.

706       layeroptions.extent = transformExtent(l.bbox, WGS84, this.map.getView().getProjection().getCode());
```

Reply:
I never had this error but your right, [ol extent](https://github.com/openlayers/openlayers/blob/v6.5.0/src/ol/extent.js#L9) only accepts for numbers.